homeassistant/components/media_player/firetv.py

```diff
@@ -68,6 +78,8 @@ def has_adb_files(value):
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the FireTV platform."""
     from firetv import FireTV
+    if DATA_KEY not in hass.data:
+        hass.data[DATA_KEY] = {}
```

Comment:
Use `dict.setdefault` instead.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/21419/commits/546d1d1b05b7f0bf1c9e9ff0490ccc374a6f2336